docs/content/what-is-a-schedule-to-start-timeout.md

```diff
@@ -10,24 +10,33 @@ tags:
 import CenteredImage from "../components/CenteredImage.js"
 import RelatedReadList from '../components/RelatedReadList.js'
 
-A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task.
+A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts (picks it up from the Task Queue) that Activity Task.
+In other words, it's a limit for how long an Activity Task can be enqueued.
+The moment that the Task is picked by the Worker from the task queue is considered to be the start of the Activity Task for the Schedule To Start timeout and associated metrics to avoid clock difference between the Server and the Worker.
```

Comment:
```suggestion
A Schedule-To-Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task (picks the Activity Task from the Task Queue).
In other words, a Schedule-To-Start Timeout is a limit on how long an Activity Task can remain in the Task Queue.
The moment the Activity Task is picked by the Worker is considered to be the start of the Activity Task Execution for the purposes of calculating the Schedule-To-Start Timeout and associated metrics.
It's designed this way in case there are clock differences between the Temporal Cluster and a Worker.
```
I personally liked it before and am advocating for you to change it back.
I do not think "It's" is too colloquial here as "It's" clearly refers to the dynamic described in the previous sentence - @djmagee what is your take on this paragraph?

Reply:
The avatar of one of the sharpest project editors I know says that Shawn is correct here. (She always insisted that I crisply define antecedents.)

I recommend changing your last sentence to something like this:

This definition of "start" avoids issues that a clock difference between the Temporal Cluster and a Worker might create.